Part2-API-Schemas/openapi.yaml

```diff
@@ -206,6 +203,53 @@ components:
         - properties: 
             derivedFrom: 
               $ref: "https://api.swaggerhub.com/domains/Plattform_i40/Part1-MetaModel-Schemas/V3.0.1#/components/schemas/Reference"
+    AssetLink:
+      properties:
+        name:
+          type: string
+          minLength: 1
+          maxLength: 64
+          pattern: "^([\\t\\n\\r -\ud7ff\ue000-\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
+        value:
+          type: string
+          minLength: 1
+          maxLength: 2000
+          pattern: "^([\\t\\n\\r -\ud7ff\ue000-\ufffd]|\\ud800[\\udc00-\\udfff]|[\\ud801-\\udbfe][\\udc00-\\udfff]|\\udbff[\\udc00-\\udfff])*$"
+        externalSubjectId:
+          "$ref": "#/components/schemas/DeprecatedReference"
+        semanticId:
+          "$ref": "#/components/schemas/DeprecatedReference"
+        supplementalSemanticIds:
+          deprecated: true
+          type: array
+          items:
+            "$ref": "#/components/schemas/DeprecatedReference"
+          minItems: 1
+      required:
+      - name
+      - value
+    DeprecatedReference:
```

Comment:
add description, e.g.
`description: A reference as value for a deprecated attribute`